CHANGELOG.md

```diff
@@ -18,6 +20,28 @@ to close the error messages box.
   - updated doc string for `assets_external_path` Dash argument to be more clear that it will allways be joined with the `assets_url_path` argument when determining the url to an external asset.
 - [#1493](https://github.com/plotly/dash/pull/1493) Fix [#1143](https://github.com/plotly/dash/issues/1143), a bug where having a file with one of several common names (test.py, code.py, org.py, etc) that imports a dash component package would make `import dash` fail with a cryptic error message asking whether you have a file named "dash.py"
 
+## DASH CORE COMPONENTS
```

Comment:
Do we normally capitalize these? I think it's probably OK just to go with Dash Core Components :slightly_smiling_face:

Reply:
https://github.com/plotly/dash/pull/1535/commits/6de7f9c749a648079f19fb3a2c331357953c4a43